install.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+. ./etc/unit.default
+
+export PATH=$UNITDIR/sbin:$PATH
+
+if [ ! -x "$UNITDIR/sbin/unitd" ]; then
+	echo "Unit is not installed at $UNITDIR"
+	exit 1
+fi
+
+if ! mkdir -p /etc/default; then
```

Comment:
POSIX sh defines `set -e`, but each implementation handles it different.
Take a look at this table: https://www.in-ulm.de/~mascheck/various/set-e/

I have been creating POSIX sh scripts for portability among UNIX systems in my current job, it works ok if you don't use any shell modifier. Because of the quirks, I think it's better to avoid using it. But if you feel it still makes sense, I can change. 

Also, I can change to use bash, if you feel it could make things easier.

Reply:
No need, was just curious and imagined that it would be something like this =P